pkg/handler/nfts_test.go

```diff
@@ -0,0 +1,228 @@
+			fields: fields{
+				entities: entityMock,
+				log:      log,
+			},
+			querySymbol:      "PH",
+			queryTokenId:     "1",
+			queryGuildId:     "",
+			wantCode:         200,
+			wantErr:          nil,
+			wantResponsePath: "testdata/get_nft_detail/200-match.json",
+		},
+		{
+			name: "query no record found",
+			fields: fields{
+				entities: entityMock,
+				log:      log,
+			},
+			querySymbol:      "qweasd",
+			queryTokenId:     "1",
+			queryGuildId:     "",
+			wantCode:         200,
+			wantErr:          nil,
+			wantResponsePath: "testdata/get_nft_detail/200-no-data.json",
+		},
+		{
+			name: "query match multiple record - no default",
+			fields: fields{
+				entities: entityMock,
+				log:      log,
+			},
+			querySymbol:      "NEKO",
+			queryTokenId:     "1",
+			queryGuildId:     "",
+			wantCode:         200,
+			wantErr:          nil,
+			wantResponsePath: "testdata/get_nft_detail/200-suggest.json",
+		},
+		{
+			name: "query match multiple record - with default",
+			fields: fields{
+				entities: entityMock,
+				log:      log,
+			},
+			querySymbol:      "NEKO",
+			queryTokenId:     "1",
+			queryGuildId:     "863278424433229854",
+			wantCode:         200,
+			wantErr:          nil,
+			wantResponsePath: "testdata/get_nft_detail/200-default.json",
+		},
+	}
+	indexerRespPH := response.IndexerGetNFTTokenDetailResponse{
+		Data: response.IndexerNFTTokenDetailData{
+			TokenID:           "1",
+			CollectionAddress: "0xb54FF1EBc9950fce19Ee9E055A382B1219f862f0",
+			Name:              "Portalhead #1",
+			Description:       "A Modern People Project.",
+			Amount:            "",
+			Image:             "https://portalheads.mypinata.cloud/ipfs/QmNip3GfiuE2rFyMrSfPaafgLyLNUiMSmCti1N8LAF5Re9/0x12791c9a355a9097e9ef4b894cbf46579ca259d281a27d0f6e6b27a00538607c.jpg",
+			ImageCDN:          "",
+			ThumbnailCDN:      "",
+			ImageContentType:  "",
+			RarityRank:        0,
+			RarityTier:        "",
+			Attributes: []response.IndexerNFTTokenAttribute{
+				{
+					CollectionAddress: "",
+					TokenId:           "",
+					TraitType:         "eyes",
+					Value:             "White POV",
+					Count:             0,
+					Rarity:            "",
+					Frequency:         "5.220%",
+				},
+				{
+					CollectionAddress: "",
+					TokenId:           "",
+					TraitType:         "face",
+					Value:             "None",
+					Count:             0,
+					Rarity:            "",
+					Frequency:         "60.140%",
+				},
+				{
+					CollectionAddress: "",
+					TokenId:           "",
+					TraitType:         "head",
+					Value:             "Woke Up Like Dis",
+					Count:             0,
+					Rarity:            "",
+					Frequency:         "2.750%",
+				},
+			},
+			Rarity: &response.IndexerNFTTokenRarity{
+				Rank:  0,
+				Score: "",
+				Total: 6771,
+			},
+			MetadataID: "",
+		},
+	}
+	indexerMock.EXPECT().GetNFTDetail("0xb54FF1EBc9950fce19Ee9E055A382B1219f862f0", "1").Return(&indexerRespPH, nil)
```

Comment:
you should move this into the each test case block.

```suggestion
	indexerMock.EXPECT().GetNFTDetail(tt.querySymbol, tt.queryTokenId).Return(&indexerRespPH, nil)
```